LayerClient/LayerClient.py

```diff
@@ -358,13 +378,15 @@ class Message(BaseLayerResponse):
     """
 
     def __init__(self, id, url, sent_at=None, sender=None,
-                 conversation=None, parts=None, recipient_status=None):
+                 conversation=None, parts=None, recipient_status=None,
+                 is_unread=False):
```

Comment:
It seems strange to default `is_unread` to False, since this would suggest all messages are read unless proven otherwise. I might change this to True.